package.json

```diff
@@ -2,14 +2,15 @@
   "name": "agenda",
   "version": "4.0.0",
   "description": "Light weight job scheduler for Node.js",
-  "main": "index.js",
+  "main": "dist/index.js",
```

Comment:
revert to "index.js"

Reply:
Done https://github.com/agenda/agenda/pull/1193/commits/4024ad2f7c37025fc366a1191c1f6ee5d8ac8fe3